tests/data/power_flow/pandapower/components/asymmetric/transformer/input.json

```diff
@@ -21,7 +21,7 @@
       "sn": 1000000.0,
       "uk": 0.1,
       "pk": 10000.0,
-      "i0": 0.001,
```

Comment:
@nitbharambe 

Can we just make `i0` and `p0` to zero? And then we can set low tolerance for comparison.